python/grass/script/core.py

```diff
@@ -1709,44 +1718,38 @@ def create_location(
     desc=None,
     overwrite=False,
 ):
-    """Create new location
+    """Create new project
 
     Raise ScriptError on error.
 
-    :param str dbase: path to GRASS database
-    :param str location: location name to create
-    :param epsg: if given create new location based on EPSG code
-    :param proj4: if given create new location based on Proj4 definition
-    :param str filename: if given create new location based on georeferenced file
-    :param str wkt: if given create new location based on WKT definition
+    :param str path: path to GRASS database or project; if path to database, project
+                     name must be specified with name parameter
+    :param str name: project name to create
+    :param epsg: if given create new project based on EPSG code
+    :param proj4: if given create new project based on Proj4 definition
+    :param str filename: if given create new project based on georeferenced file
+    :param str wkt: if given create new project based on WKT definition
                     (can be path to PRJ file or WKT string)
     :param datum: GRASS format datum code
     :param datum_trans: datum transformation parameters (used for epsg and proj4)
-    :param desc: description of the location (creates MYNAME file)
-    :param bool overwrite: True to overwrite location if exists(WARNING:
-                           ALL DATA from existing location ARE DELETED!)
+    :param desc: description of the project (creates MYNAME file)
+    :param bool overwrite: True to overwrite project if exists (WARNING:
+                           ALL DATA from existing project ARE DELETED!)
     """
-    # create dbase if not exists
-    if not os.path.exists(dbase):
-        os.mkdir(dbase)
+    # Support ~ in the path for user home directory.
+    path = Path(path).expanduser()
```

Comment:
Not needed, done in `resolve_mapset_path`, no?

Reply:
You're right. Removed this line in latest commit.